packages/contracts/contracts/ArtonomousCurvedBond.sol

```diff
@@ -1,14 +1,16 @@
 pragma solidity ^0.4.24; //solhint-disable-line compiler-fixed
 
+import './EthBondingCurvedToken.sol';
 
-contract ArtonomousCurvedBond is EthPolynomialCurvedToken {
```

Comment:
The compiler didn't pass this for me. I'm not an expert but I replaced it with what is in the next line. Please let me know if I need to change this to something else. I'm a solidity noob.

Reply:
This is right, yes.

Reply:
Compiler now passes this 🤷‍♂️